infra/gcb/templates/src/build-status/build-status.html

```diff
@@ -216,8 +322,25 @@
         let timezone = ds.substring(ds.indexOf("("));
         return date.toLocaleString() + " " + timezone;
       }
-    };
 
-    window.customElements.define(BuildStatus.is, BuildStatus);
+      pad(n) {
+        {return n<10 ? '0'+n : n;}
+
+      }
+
+      getLocalDate(str) {
+        let date = new Date(str);
+        var day = this.pad(date.getDate());
+        var month = this.pad(date.getMonth() + 1);
```

Comment:
why is this `+ 1` ?

Reply:
It gives month from 0 to 11

Reply:
Just use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString ?